python/tests/lang/test_context_manager.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+from aac.lang.language_context import LanguageContext
+
+from aac.lang.active_context_lifecycle_manager import get_active_context
+from aac.plugins.plugin_manager import get_plugin_definitions
+from aac.spec import get_aac_spec
+
+
+class TestContextManager(TestCase):
```

Comment:
I think we should rename this class and the file to reflect the updated filename in `src/`.

```suggestion
class TestActiveContextLifecycleManager(TestCase):
```